docs/CONTRIBUTING.md

```diff
@@ -1,21 +1,271 @@
+
+1. Open the file in your favorite text editor, and add the following minimum content:
+
+   ```yaml
+   name: "My feature"
+   description: "A short description of the feature."
+   spec: https://urlofthespec.com
+   ```
+
+   For guidance, see the [Names](./guidelines.md#names) and [Descriptions](./guidelines.md#descriptions) sections of the feature guidelines.
+
+1. Optionally add a `group` field to the feature.
+
+   The `group` field is used to categorize features into groups. For example, the Async Clipboard API feature is in the `Clipboard` group. Groups are maintained in the [`groups`](https://github.com/web-platform-dx/web-features/tree/main/groups) directory, and each group is a YAML file.
+
+   * If one of the existing groups fits your feature, add the `group` field to your feature, and set it to the name of the group.
+   * If none of the existing groups fit your feature, but you believe a new group should be greated, then create a new group file in the `groups` directory, and add the feature to that group.
+
+1. Set the baseline status and browser compatibility data of the feature. This is the most important and difficult step of authoring a feature, which is documented in a separate section. See [Set the status of a feature](#set-the-status-of-a-feature).
+
+### Create a new feature from a draft feature
+
+To start from an existing draft feature:
+
+1. Go through the steps in [Fork the repository and set up your local environment](#fork-the-repository-and-set-up-your-local-environment).
+
+1. Review the existing draft features by looking at the YAML files in the [`features/draft`](https://github.com/web-platform-dx/web-features/tree/main/features/draft) directory and sub-directories.
+
+1. Find a draft that you want to work on.
+
+1. Move the file to the `features` directory.
+
+1. Remove the `draft_date` field from the file.
+
+1. Review the feature ID, name, description, and spec fields.
+
+1. Either review, correct, or set the baseline status and browser compatibility data of the feature. This is the most important and difficult step of authoring a feature, which is documented in a separate section. See [Set the status of a feature](#set-the-status-of-a-feature).
+
+### Set the status of a feature
+
+After you've prepared your new feature file with its unique ID, `name`, `description`, and `spec` fields, you must provide the necessary information for this feature's status to be calculated.
+
+A feature's status consists of:
+
+* Whether the feature is considered baseline or not, and which level of baseline it has achieved:
+
+  * Baseline _low_ means that the feature is now available on all browsers listed in the core browser set.
+  * Baseline _high_ means that the feature has been available on all browsers listed in the core browser set for long enough that it's considered _widely_ available.
+
+  For more information about baseline, see [Baseline](./baseline.md).
```

Comment:
I suppose you don't want to duplicate the baseline high definition here as it could then become out of sync. It may be worth emphasizing that this is where the notion of "long enough" is captured:

```suggestion
  For more information about baseline and the actual definition of _long enough_, see [Baseline](./baseline.md).
```